app/assets/javascripts/host_edit.js

```diff
@@ -793,7 +793,7 @@ function interface_subnet_selected(element, ip_field, skip_mac) {
   preserve_selected_options($(element));
 
   var subnet_id = $(element).val();
-  if (subnet_id == '') return;
+  if (subnet_id == '' || subnet_id == 'No subnets') return;
```

Comment:
Won't this break in translations? I think it would be cleaner to store the ID as a data attribute, similar to how we store the VLAN ID so you can avoid human readable strings.

Reply:
I agree, I will fix it.